view/frontend/web/js/view/payment/method-renderer/tns-hpf.js

```diff
@@ -82,7 +87,8 @@ define(
                         'adapterLoaded',
                         'creditCardExpYear',
                         'creditCardExpMonth',
-                        'buttonTitle'
+                        'buttonTitle',
+                        'showCcForm',
```

Comment:
Extraneous comma?

Reply:
Thank you for mentioning it! I will fix the issue and update the Pull Request.